app/data/models.py

```diff
@@ -34,7 +35,45 @@ class ActionTypes(object):
             (UPDATE, 'Update'),
             (DELETE, 'Delete')
         )
+
         @classmethod
         def as_list(cls):
             return [cls.CREATE, cls.UPDATE, cls.DELETE]
+
     action = models.CharField(max_length=6, choices=ActionTypes.choices)
+
+
+class CeleryJob(models.Model):
```

Comment:
Cool, that makes sense; I think there will also eventually be a report generation feature which would likely need to be asynchronous, but probably wouldn't need to be tracked over the long term. Not sure about blackspots; we might be able to get all the information we need from the blackspot and blackspotset models themselves.